AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -907,4 +899,15 @@ - (void)rangeController:(ASRangeController *)rangeController didDeleteSectionsAt
   }
 }
 
+#pragma mark - ASCellNodeDelegate
+
+- (void)node:(ASCellNode *)node didRelayoutWithSuggestedAnimation:(ASCellNodeAnimation)animation
+{
+  ASDisplayNodeAssertMainThread();
+  NSIndexPath *indexPath = [self indexPathForNode:node];
+  if (indexPath != nil) {
+    [super reloadItemsAtIndexPaths:@[indexPath]];
```

Comment:
I wonder if there's a way to coalesce these without limiting the ability to wrap in animation blocks.  Maybe it's not an issue, but do you see visually sequential loading / flashing if you set multiple cells as needing layout?  Even if there were a longer delay due to main thread measurement, it would be best for all of those to happen together.